src/toffy/rosetta.py

```diff
@@ -301,7 +301,7 @@ def compensate_image_data(
 
     # loop over each set of FOVs in the batch
     for i in range(0, len(fovs), batch_size):
-        print("Processing image {}".format(i + 1))
+        print(f"Processing FOV {fovs[i]}")
```

Comment:
Can probably just be "Processing fov-1-scan-1" instead of "Processing FOV fov-1-scan-1".